src/io/flutter/devtools/DevToolsManager.java

```diff
@@ -166,6 +173,7 @@ private OSProcessHandler getProcessHandlerForBazel() {
       handler = new OSProcessHandler(commandLine);
     }
     catch (ExecutionException e) {
+      FlutterUtils.warn(LOG, e);
       e.printStackTrace();
```

Comment:
I'd remove the `e.printStackTrace();` call here.

Reply:
Done.